mars/tensor/execution/datasource.py

```diff
@@ -210,6 +210,10 @@ def _scalar(ctx, chunk):
     ctx[chunk.key] = _create_array(chunk.op)('asarray', chunk.op.data)
 
 
+def _dataframe_to_tensor(ctx, chunk):
+    ctx[chunk.key] = ctx[chunk.inputs[0].key].values
```

Comment:
Pandas recommend using `DataFrame.to_numpy()` (https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.values.html#pandas.DataFrame.values), but it is new in version 0.24.0, maybe we need a util to be compatible.

Reply:
hi, I import 'packaging' to compare version. Is that OK? Please see my latest commit

Reply:
Just use try except.